resources.md

```diff
@@ -69,3 +63,15 @@ There are thousands of books on programming in Python. This list focuses on titl
 - [Think Stats!](https://greenteapress.com/wp/think-stats-2e/) and [Think Bayes!](http://greenteapress.com/wp/think-bayes/) by Allen Downey. These books are aimed at teaching probability & statistics and Bayesian statistics, using Python rather than traditional mathematical expressions to make the concepts concrete. Recommended for more advanced beginners with an interest in the mathematical underpinnings of statistics. The website for both books has free PDF and HTML versions.
 
 - [Python Tricks: The Book](https://realpython.com/products/python-tricks-book/) by Dan Bader. This is a great book for those who are comfortable with the basics of Python and would like to learn some more advanced tricks, such as decorators, abstract base classes, named tuples, and more. The author runs the [Real Python site](https://realpython.com/) which has a lot of in-depth Python tutorials. A short [PDF sample](https://static.realpython.com/python-tricks-book/Python%20Tricks%20Sample.pdf) is available. Libraries: [MLN](https://find.minlib.net/iii/encore/record/C__Rb3739402), [BPL](https://bpl.bibliocommons.com/item/show/12035076981)
+
+## Newsletters, mailing lists, discussion boards
+
+If you want to learn more about a language, listening to other people talk about it can be helpful. Discussion forums can give you new ideas and lead you into parts of the language that you wouldn't have otherwise discovered, and expose you to the ways other people think about and use the language.
+
+- [PyHerald](https://pyherald.com/) - A daily summary of what's happening on various Python forums, apparently sourced from official and unofficial Python mailing lists and discussion boards. Probably a good read for people who want to know what's going on in the Python world but don't have time to follow all of the lists themselves.
+
+- [python-ideas](https://mail.python.org/mailman3/lists/python-ideas.python.org/) - This list is mostly about proposals for future development of the Python language. Sometimes a bit contentious, sometimes a bit technical. Volume is typically moderate to low, but can surge if a particularly fraught proposal is made.
```

Comment:
From the description, and what I've heard about python-ideas on newsletters and podcasts, it feels out of the scope of this page and its intended audience:

> This page lists a number of places you can go for ideas about what programs to write to develop your Python skills.

After a quick scan, I think the same might be true of PyHerald.

There are lots of Python-related newsletters, mailing lists, discussion boards, and podcasts. Are they useful to add to this page? How do we decide what makes the cut? I already feel like I'm bike-shedding. Maybe there's already a list of them somewhere, that we could link to as an additional resource?

Reply:
My thinking in adding them was that they're good sources of curiosity for intermediate python programmers. They're probably not for everybody, but that's true of everything. 

Is there a quick fix you'd suggest to make this addition make more sense to you? If so, I'm happy to do that. If not, I'm fine with dropping this.

Reply:
I'm inclined to drop it for now.